processor/src/main/resources/org/mapstruct/ap/internal/model/AnnotationMapperReference.ftl

```diff
@@ -19,7 +19,4 @@
      limitations under the License.
 
 -->
-<#list annotations as annotation>
-<#nt><@includeModel object=annotation/>
-</#list>
 private <@includeModel object=type/> ${variableName};
```

Comment:
This can only be final, if there is no default public constructor, as in CDI. I will change, so that the fields are final, if no public constructor is needed